$MirOS: ports/www/dillo/v2/patches/patch-src_auth_c,v 1.2 2010/02/21 20:08:42 bsiegert dead $
--- src/auth.c.orig	Sun Jun  7 03:40:26 2009
+++ src/auth.c	Mon Aug 24 16:22:10 2009
@@ -228,7 +228,7 @@ static int Auth_parse_token_value(AuthPa
    set_realm =
       auth_parse->realm == NULL &&
       dStrncasecmp(realm_token,token,token_size) == 0 &&
-      strlen(realm_token) == token_size;
+      strlen(realm_token) == (size_t)token_size;
 
    return Auth_parse_quoted_string(auth_parse, set_realm, auth);
 }
